src/main/java/com/greghaskins/spectrum/ConstructorBlock.java

```diff
@@ -16,14 +18,19 @@ public void run() throws Throwable {
     try {
       final Constructor<?> constructor = this.klass.getDeclaredConstructor();
       constructor.setAccessible(true);
-      constructor.newInstance();
+      testObject = constructor.newInstance();
```

Comment:
We'll need this soon... I'm planning a cheat version of the rule stuff where you can use the test class the old way if you like and it just kinda works...